pom.xml

```diff
@@ -11,16 +11,16 @@
   <name>auth-code-notification</name>
   <properties>
     <java.version>11</java.version>
-    <spring-boot-dependencies.version>2.6.1</spring-boot-dependencies.version>
-    <spring-boot-maven-plugin.version>2.6.1</spring-boot-maven-plugin.version>
+    <spring-boot-dependencies.version>2.6.15</spring-boot-dependencies.version>
+    <spring-boot-maven-plugin.version>2.6.15</spring-boot-maven-plugin.version>
     <log4j.version>2.17.2</log4j.version>
     <avro.version>1.11.1</avro.version>
     <ch-kafka.version>1.4.8</ch-kafka.version>
     <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
     <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
     <structured-logging.version>1.9.19</structured-logging.version>
-    <api-sdk-manager-java-library.version>1.0.4</api-sdk-manager-java-library.version>
-    <json.version>20230227</json.version>
+    <api-sdk-manager-java-library.version>1.0.7</api-sdk-manager-java-library.version>
+    <json.version>20230618</json.version>
```

Comment:
The JSON CVE still seems to be present but commit message suggests it should be gone:

`json-20230618.jar (pkg:maven/org.json/json@20230618, cpe:2.3:a:json-java_project:json-java:*:*:*:*:*:*:*:*) : CVE-2022-45688`

Reply:
Yes - apparently that one is still open with the library, see https://nvd.nist.gov/vuln/detail/CVE-2022-45688. Shall I update commit message, or supress this in another commit and note it?

Reply:
I think PR commit message/description is now fine. Clears this up for me. Thanks.